### PR TITLE
Add new transit features

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to #469

This is still a work in progress, but I wanted to share what I had thus far in case you noticed something already. I was originally going to just move ahead with trying to get it to work with manual sql functions, but now I think it'll actually be easier to transition to a csv for transit as well because the kind isn't being set properly. The logic is the same as with the pois layer, so it should be easy enough to just copy over the relevant csv rows for transit and query those for lines/polygons.

@zerebubuth could you review please?

### CODE REVIEW

File: data/migrations/v0.9.0-polygons.sql [hunk 1]
```diff
@@ -4,6 +4,13 @@ UPDATE planet_osm_polygon SET
     "disused" <> 'no' OR
     "railway" = 'station';
 
+UPDATE planet_osm_polygon SET
```
Comment (zerebubuth) [inline data/migrations/v0.9.0-polygons.sql]: Does this need to have the same `ALTER TABLE` as `apply-planet_osm_polygon.sql` to add the column?

Comment (rmarianski) [inline data/migrations/v0.9.0-polygons.sql]: Nice catch! Updated in ae52cae.

**Comment by rmarianski** [comment]
I transitioned to using the a csv file for `transit` as well as mentioned in the description and rebased the commits.

Something I just realized is that the score calculation is now only done for the `pois` layer. Is that something that we'll want to do in the `transit` layer too? If so, across both line and polygon tables?

**Comment by nvkelso** [comment]
CSV is legible, nice work!

I believe we only calculate the score for `tile_kind_rank` on `kind=station` features. We might want to extend that in the future to more kinds, but out of scope for this release I think.

We should only ever calculate that for POI layer content (not transit layer content which should just be lines and polygons).

**Comment by zerebubuth** [comment]
:+1: looks good. :zap: on the macro for the transit SQL - that looks great.

**Pull request merged**